freecodecamp.conf.json

```diff
@@ -1,7 +1,8 @@
 {
   "path": ".",
+  "version": "0.1.0",
   "scripts": {
-    "develop-course": "NODE_ENV=development nodemon --watch ./tooling/ --ignore ./config/ .freeCodeCamp/tooling/server.js",
+    "develop-course": "NODE_ENV=development node ./.freeCodeCamp/tooling/server.js",
```

Comment:
If run and develop are the same, do we need both? If we aren't planning on differentiating it in the future, I would maybe get rid of it. Doesn't need to be now - your call.

Reply:
They are not the same. One is `NODE_ENV=development` and the other is `NODE_ENV=production`

Reply:
k